### ISSUE TITLE
Advanced examples

### ISSUE THREAD

**Issue by mehrhardt** [odlgroup/odl#1209]
This issue is to unify the discussion on advanced examples as we were already having in #1101 and possibly other places, too.

The big problem here is that we would like to have two kinds of examples. One which is really introductory and helps people get going with ODL and others which show more advanced functionality of ODL and its usage for "real" problems. It should be obvious to the user which examples are which and it should be easy to navigate in each of these. Thus it probably makes sense to have two different folders for these.

**Comment by adler-j** [comment]
My suggestion would perhaps be to put the advanced examples in a sub-folder to the "regular examples" folder, would that be OK?

**Comment by mehrhardt** [comment]
Sounds good to me. Would you then also have the same tree structure in there?

**Comment by adler-j** [comment]
Well I would prefer

```
examples/
├── tomo/
│   ├── filtered_backprojection_cone_2d.py
│   ├── filtered_backprojection_cone_3d.py
│   ├── ...
│   └── advanced/
│       ├── super_advanced_example.py
│       └── ...
├── trafos/
│   └── ...
└── ...
```

**Comment by kohr-h** [comment]
Exactly, that's what I had in mind, too. Users will first see the "regular" ones on top level and could go to the advanced ones by diving into the subfolder.
